test/integration/drivers/virtualbox/upgrade.bats

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bats
+
+load ../../helpers
+
+export OLD_ISO_URL="https://github.com/boot2docker/boot2docker/releases/download/v1.4.1/boot2docker.iso"
+
+@test "$DRIVER: create for upgrade" {
+  run machine create -d virtualbox --virtualbox-boot2docker-url $OLD_ISO_URL $NAME
+}
+
+@test "$DRIVER: upgrade" {
+  run machine upgrade $NAME
+  echo ${output}
+  [ "$status" -eq 0  ]
+}
+
+@test "$DRIVER: upgrade is correct version" {
+  SERVER_VERSION=$(docker $(machine config $NAME) version | grep 'Server version' | awk '{ print $3; }')
+  [[ "$SERVER_VERSION" != "1.4.1" ]]
```

Comment:
We just load helpers in relative form, such as `load ../helpers`. bats already takes care of the rest